docs/source/models/intervention_models/intrapartum/azithromycin_intervention.rst

```diff
@@ -0,0 +1,265 @@
+placeholder values come from three data sources, all for Ethiopia, all identified by the Health Systems team at IHME: the 2016 
+Ethiopia EmONC Final Report found 39.2% of BEmONC facilities and 90.4% of CEmONC facilities have azithromycin; the 2016 SARA Report 
+found 68.4% of BEmONC facilities and 90.5% of CEmONC facilities have azithromycin; the 2018 SARA Report found 75.9% of BEmONC 
+facilities and 98.0% of CEmONC facilities have azithromycin. While we plan a data strategy to fill the gaps we have used a simple 
+average.
+
+.. list-table:: Baseline Coverage of intrapartum azithromycin
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Birth Facility
+    - Coverage Mean (%)
+    - Coverage Distribution (%)
+    - Notes
+  * - Home Birth
+    - 5
+    - :math:`\text{Uniform}(0,10)`
+    - Assumption; need to investigate data sources for care seeking those giving birth outside of the hospital system 
+  * - BEmONC Facilities
+    - 61.2
+    - :math:`\text{Uniform}(39.2,75.9)`
+    - placeholder value based on three data points 
+  * - CEmONC Facilities
+    - 93.0
+    - :math:`\text{Uniform}(90.4,98.0)`
+    - placeholder value based on three data points 
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+This intervention requires adding an attribute to all simulants to specify if a pregnant person has access to a facility with access to azithromycin.  We will track if a simulant has access to azithromycin 
+and the model will have different incidence rates for sepsis for individuals with and without access to azithromycin (implemented with a slightly confusing application of our ``Risk`` and ``RiskEffect`` 
+components from ``vivarium_public_health``).
+
+The ``Risk`` component adds an attribute to each simulant indicating whether the simulant has access to azithromycin during the intrapartum period, which we assume will be closely 
+related to the facility choice during birth, i.e. home births have much lower access than in-facility births, and births in BEmONC facilities have lower access than CEmONC 
+facilities.
+
+To make this work naturally with the ``RiskEffect`` component, it is best to think of the risk as "lack of access to azithromycin".  With this framing, the ``RiskEffect`` 
+component requires data on (1) the relative risk of sepsis incidence for people with lack of access to azithromycin, and (2) the population attributable fraction (PAF) of sepsis 
+due to lack of access to azithromycin.  We will use the decision tree below to find the probability of sepsis incidence with and without access to azithromycin that are logically 
+consistent with the baseline delivery facility rates and baseline azithromycin coverage.
+
+In Vivarium, this risk effect will modify the sepsis incidence pipeline, resulting in 
+
+.. math::
+
+   \text{IR}_i^\text{sepsis} = \text{IR}^\text{sepsis}_{\text{BW}_i, \text{GA}_i} \cdot (1 - \text{PAF}_\text{no azithromycin}) \cdot \text{RR}_i^\text{no azithromycin}
```

Comment:
I am confused about why we have BW and GA here. Maternal sepsis rates should not vary by LBWSG, but neonatal sepsis rates will. Maybe let's add "maternal" in front of everywhere we say sepsis to make sure this is super clear because I definitely think it will be a common source of confusion!!

Unless I am misinterpreting something and we actually do want to affect neonatal sepsis rates? In that case I think we will need to update the affected measure to be CSMR rather than IR?